src/conn/conn_layered.c

```diff
@@ -1228,6 +1269,7 @@ __layered_drain_ingest_table(WT_SESSION_IMPL *session, WT_LAYERED_TABLE_MANAGER_
             upd = NULL;
         }
     }
+    WT_ERR(__layered_clear_ingest_table(session, entry->ingest_uri));
```

Comment:
It's better to call this outside this function. So we have a function that drains the table and another function that truncates it.

Reply:
I disagree because to me, "drain" implies the original is gone. But I'm happy to be outvoted or hear differing opinions.

Reply:
What if we rename ...drain.... to be `__layered_copy_ingest_table`, then you can move clear function outside it.

Reply:
I think this is a good idea.

Reply:
Thanks for the suggestions - `e04ca09` does this, as well as the other thread about not opening an internal session.